pkg/apis/submariner.io/v1/string_test.go

```diff
@@ -0,0 +1,38 @@
+package v1
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const expectedString = "{\"cluster_id\":\"cluster-id\",\"cable_name\":\"cable-1\",\"hostname\":" +
+	"\"\",\"subnets\":[\"10.0.0.0/24\",\"172.0.0.0/24\"],\"private_ip\":\"1.1.1.1\"," +
+	"\"public_ip\":\"\",\"nat_enabled\":false,\"backend\":\"\"}"
+
+var _ = Describe("API v1", func() {
+	Describe("Endpoint String representation", func() {
+		When("Called", func() {
+			It("Should return a human readable string", func() {
```

Comment:
I think you can combine the `Describe` and `When`, ie

When("Endpoint String function is called", func() {
    It("should return a human readable string", ...
})